faqs.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>FAQs</title>
+</head>
+<body>
+    <header>
+        <div>
+          <img src="/images/logo.png" alt="logo" width="250" height="auto" />
+        </div>
+    
+        <style>
+          body {
+            font-family: Arial, sans-serif;
+            margin: 0;
+          }
+    
+          nav {
+            background-color: #333;
+            overflow: hidden;
+          }
+    
+          nav a {
+            display: inline-block;
+            color: white;
+            text-align: center;
+            padding: 14px 16px;
+            text-decoration: none;
+          }
+    
+          nav a:hover {
+            background-color: #ddd;
+            color: black;
+          }
+        </style>
```

Comment:
💄 Good use of inline CSS, but consider externalizing ( having it on a separate CSS file)  for cleaner code. Keep it up!